components/renku_data_services/notebooks/api.spec.yaml

```diff
@@ -865,6 +886,8 @@ components:
           type: integer
         cloudstorage:
           $ref: "#/components/schemas/SessionCloudStoragePostList"
+        env_variables:
```

Comment:
suggestion: If this is only an override (not an upsert) than the name of this variable should have "override" in the name imo. As it stands right now it makes me think it is an upsert.